CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Change Log
 
+## 0.26.0 - 2025-04-02
+This release enables the creation of web apps with a unique default hostname to avoid a high-severity threat of subdomain takeover.
+The web app basic create will now default to a `tenant` scoped hostname uniqueness, while the web app advanced create will allow
+the choice of either `tenant` (tenant reuse) or `global` (no reuse) level domain name scope.  For full context on this topic,
+please refer to this [post](https://techcommunity.microsoft.com/blog/appsonazureblog/public-preview-creating-web-app-with-a-unique-default-hostname/4156353).
+
+### Added
+* [[2703](https://github.com/microsoft/vscode-azureappservice/pull/2703)] Add support for unique default hostnames when creating a new web app.
```

Comment:
Should we add a changed section that tells users that we have changed the order of steps in creation? I'm sure that they'll figure it out, but I could see someone thinking that it's a bug initially.

Reply:
I'll add a note